cmd/prepare-storage.go

```diff
@@ -169,28 +168,24 @@ func prepForInitXL(firstDisk bool, sErrs []error, diskCount int) InitActions {
 		return WaitForFormatting
 	}
 
-	// Total disks unformatted are in quorum verify if we have some offline disks.
-	if disksUnformatted >= quorum {
-		// Some disks offline and some disks unformatted, wait for all of them to come online.
-		if disksUnformatted+disksFormatted+disksOffline == diskCount {
-			return WaitForAll
-		}
-
-		// Some disks possibly corrupted and too many unformatted disks.
-		return Abort
-	}
-
 	// Already formatted and in quorum, proceed to initialization of object layer.
 	if disksFormatted >= readQuorum {
 		if disksFormatted+disksOffline == diskCount {
 			return InitObjectLayer
 		}
 
 		// Some of the formatted disks are possibly corrupted or unformatted, heal them.
-		return WaitForHeal
+		return SuggestToHeal
+	}
+
+	// Some disks unformatted, Some disks formatted and some disks are offline.
```

Comment:
Can make the comment also say that we don't have quorum here that's why this is undecisive state

Reply:
Done